instance/templates/instance/ansible/vars.yml

```diff
@@ -44,16 +44,16 @@ SECURITY_UPDATE_ALL_PACKAGES: false
 SECURITY_UPGRADE_ON_ANSIBLE: true
 
 # Repositories URLs
-edx_ansible_source_repo: '{{ instance.ansible_source_repo_url }}'
-edx_platform_repo: '{{ instance.repository_url }}'
+edx_ansible_source_repo: '{{ instance.configuration_source_repo_url }}'
+edx_platform_repo: '{{ instance.edx_platform_repository_url }}'
 
 # Pin down dependencies to specific (known to be compatible) commits.
-edx_platform_version: '{{ instance.commit_id }}'
+edx_platform_version: '{{ instance.edx_platform_commit }}'
 configuration_version: '{{ instance.configuration_version }}'
-forum_version: '{{ instance.forum_version }}'
-notifier_version: '{{ instance.notifier_version }}'
-xqueue_version: '{{ instance.xqueue_version }}'
-certs_version: '{{ instance.certs_version }}'
+forum_version: '{{ instance.openedx_release }}'
+notifier_version: '{{ instance.openedx_release }}'
+xqueue_version: '{{ instance.openedx_release }}'
+certs_version: '{{ instance.openedx_release }}'
```

Comment:
Yes, I checked and all four were always set to the same value on all our existing sandboxes. Some were master, some dogwood, some cypress, but these four fields always matched.